composer.json

```diff
@@ -0,0 +1,36 @@
+
+{
+    "name": "chameleon-system/upgrade-helper",
+    "type": "symfony-bundle",
+    "description": "A Symfony bundle for chameleon-system that helps upgrading. Currently supported is an upgrade to 7.1",
+    "keywords": [
+        "check",
+        "upgrade",
+        "administration",
+        "bundle",
+        "Symfony"
+    ],
+    "homepage": "http://www.chameleon-system.de",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "ESONO AG",
+            "email": "info@chameleonsystem.com",
+            "homepage": "https://www.chameleonsystem.com",
+            "role": "Company"
+        }
+    ],
+    "require": {
```

Comment:
Dependencies are missing here - this package depends on at least `symfony/container` and `symfony/finder`. Also, if this package is to be installed as a dependency of a project, then `require-dev` of the subpackage is not installed with it - so `nikic/php-parser` should probably be moved into `require` as well.

Reply:
container is part of symfony/dependency-injection, which is already required. But you are right about finder. 
And yes, I totally forgot about the fact, that dev dependencies won't be installed if this bundle is required. I moved everything to ' require' except phpunit

Reply:
:ok: